### ISSUE TITLE
Lifting of embargo

### ISSUE THREAD

**Issue by lnielsen** [inveniosoftware/invenio-rdm-records#871]
Reports on Discord chat channel seems to indicate that the lifting of embargo might not work as expected.

To verify:
1) The task and embargo date is well aligned (e.g we store UTC and the task may be run in any timezone).

2) The embargo date should be lifted at 00:00 on the embargo date.

**Comment by ppanero** [comment]
I think the problem comes from the actual ES query (not sure how is passing the tests...). We are querying *from \* to now*, see [here](https://github.com/inveniosoftware/invenio-rdm-records/blob/master/invenio_rdm_records/services/services.py#L86)

From [ES docs](https://www.elastic.co/guide/en/elasticsearch/reference/current/search-aggregations-bucket-daterange-aggregation.html): 

> Note that this aggregation includes the from value and excludes the to value for each range.)

Therefore, I think the embargo will be lifted one day later. I created a record in rdm demo site with embargo lift tomorrow (2nd of march 2022)

https://inveniordm.web.cern.ch/records/cjbe6-xsz98

Slow testing to see what happens 😅

**Comment by ppanero** [comment]
TL;DR: Cannot reproduce
